### ISSUE TITLE
Notifier doesn't send all new topics, only the first one

### ISSUE THREAD

**Issue by mtomas7** [SimpleMachines/SMF#4869]
After a lengthy discussion ( https://www.simplemachines.org/community/index.php?topic=561369.0 ) it seems that when it comes about notification of New Topics an the board, user get's only 1st New Topic Notification.

Even if user selected under Notification settings to receive notification 'Straight away', vs 'Straight away (but only for the first unread message)', there would be only 1st New Topic announcement sent.

Possible resolution could also be to create separate setting to allow admins enable sending all New Topic announcement for the board for which user has subscription.

**Comment by mtomas7** [comment]
I looked at CreatePost-Notify.php and it looks that code only foresees this one action: notify once. Could anyone suggest how to change the code, so all new topics would be notified?

```
// A new topic in a watched board then?
elseif ($type == 'topic')
{
	$pref = !empty($prefs[$member]['board_notify_' . $topicOptions['board']]) ? $prefs[$member]['board_notify_' . $topicOptions['board']] : (!empty($prefs[$member]['board_notify']) ? $prefs[$member]['board_notify'] : 0);

	$content_type = 'board';

	$message_type = !empty($frequency) ? 'notify_boards_once' : 'notify_boards';
	if (!empty($prefs[$member]['msg_receive_body']))
		$message_type .= '_body';
}
```

**Comment by mtomas7** [comment]
I asked for a paid help:

https://www.simplemachines.org/community/index.php?topic=561416.0

I hope someone will be able to fix this bug.

**Comment by mtomas7** [comment]
Hi, guys, I really need your help  :-\

Nobody responded yet to my post here or on the github, to take this job even for the fee. I also privately contacted several SMF devs, but they don't want to take the job too. I have the feeling that it is much deeper burred in the core, so only experienced SMF devs can actually work on it.

Can anyone recommend SMF developers, so I could try to reach them out, hoping they would have reasonable price tag and that they would be able to make this fix/improvement, so whole SMF community would benefit if it would be implemented with admin/user option.

Thank you!

**Comment by Oldiesmann** [comment]
@Arantor wrote the code so he's the one who could probably help with this.

**Comment by mtomas7** [comment]
@Oldiesmann Thanks for the reply! @Arantor  agreed to make the change if he would know that it will be accepted by the SMF team: https://www.simplemachines.org/community/index.php?topic=561369.msg3983980#msg3983980

How we can initiate the discussion, for this fix to be accepted?

Thank you!

**Comment by mtomas7** [comment]
Thankfully, there is a fix for this problem!

https://www.simplemachines.org/community/index.php?topic=561369.msg3984037#msg3984037

**Comment by Sesquipedalian** [comment]
Apparently solved by #4924 and/or #4978.
